defaults/network-ents.go

```diff
@@ -134,7 +134,7 @@ func networkProxyEntitlementEnforce(profile secprofile.Profile) (secprofile.Prof
 	capsToRemove := []types.Capability{CapNetAdmin}
 	ociProfile.RemoveCaps(capsToRemove...)
 
-	capsToAdd := []types.Capability{CapNetBroadcast, CapNetRaw, CapNetBindService}
```

Comment:
This is a typo, a proxy needs the ability to broadcast and listen to multicast. I didn't get the opportunity to test this specific entitlement properly.

Note: `network.proxy` will be a complicated one anyway.